app.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+
+const db = require('./database/connection');
+const registration = require('./routes/registration/registration');
+const tasks = require('./routes/tasks/tasks');
+const users = require('./routes/users/users');
+
+const app = express();
+const port = process.env.PORT | 5000;
```

Comment:
make sure you use two pipes ie `||` what you're doing is accidentally a bitwise operation.